src/database/UIDatabase.js

```diff
@@ -13,10 +13,14 @@ export class UIDatabase {
       case 'Supplier':
         results = results.filtered('isVisible == true AND isSupplier == true');
         break;
+      case 'mSupplyStore':
```

Comment:
There is a principle in the react community of 'single return'. We try to follow where it is easy